packages/melange.js/js.ml

```diff
@@ -200,196 +200,254 @@ module Exn = struct
   let raiseUriError str = raise (UriError str)
 end
 
-module Array2 = struct
-  type 'a t = 'a array
+module Array : sig
   (** JavaScript Array API *)
 
+  type 'a t = 'a array
   type 'a array_like
 
-  let from _ = notImplemented "Js.Array2" "from"
-  let fromMap _ _ = notImplemented "Js.Array2" "fromMap"
+  val from : 'a array_like -> 'a array
```

Comment:
I know this means we're duplicating signatures here and in mli. But it's helpful to generate the `mli` from melange original repo module (e.g. generate `Js_array.mli` from `Js_array.ml`), replace `external` with `val` and just paste it here, to make sure both interfaces stay in sync.